### ISSUE TITLE
Loss and mAP chart in YOLOv4 how imporove

### ISSUE THREAD

**Issue by interface79** [pjreddie/darknet#2587]
i train 3 class in darknet at gpu 4 gb nvidia gtx 1650 i edit cfg files
batch=16
subdivisions=64
width=448
height=448 channels=3
learning_rate=0.0013
burn_in=1000
max_batches = 6000
policy=steps
steps=4800,5400
scales=.1,.1
below is the loss graph at the end of the training
![chart_yolov4](https://user-images.githubusercontent.com/26420935/216381444-8809eceb-beb6-4028-a50c-1724b4afdcce.png)

6000: 4.695510, 6.366718 avg loss, 0.000013 rate, 28.772000 seconds, 384000 images, 0.516126 hours left

I tested the yolov4_last.weights file and got the following result
![Screenshot_3](https://user-images.githubusercontent.com/26420935/216381726-e06eac0f-4c06-443d-b4c1-48da226eb0f5.jpg)
but avg loss is6.366718 and very high how can i improve this and

**Comment by kohjiaxuan** [comment]
YOLOv4 does poorly when different classes have 100% IOU overlap in image (in this case Person and Helmet). Use a different model or retrain 2 YOLOv4 models for each class to ensure no overlap of bounding boxes
